index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Abdullah Sakhizada</title>
+    <link rel="stylesheet" type="text/css" href="assets/css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;800&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+    <div class="home-page">
+        <header>
+            <nav id="nav">
+                <a class="logo" href="#"><img src="assets/images/microverse.jfif" alt="my logo" /></a>
+
+                <nav class="hamburger"><img src="assets/images/Union.svg" alt="Menu"></nav>
+                <ul class="internal-link">
+                    <li><a href="">Home</a></li>
+                    <li><a href="">About</a></li>
+                    <li><a href=""> Contact </a></li>
+                </ul>
+
+            </nav>
+        </header>
+        <section id="home-header">
+            <h5 class="description">Hello Microverse <br>Abdullah Sakhizada <br>
```

Comment:
- [ ] Please, consider giving your main title the `h1` tag. `h1` tag is the one that comes first and shows search engines what content can be expected on the rest of the page